src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsAdDriverForcedStatusOasis.java

```diff
@@ -8,19 +8,21 @@
 
 import java.util.List;
 
-public class TestAdDriverForcedStatus extends TemplateNoFirstLoad {
+public class TestAdsAdDriverForcedStatusOasis extends TemplateNoFirstLoad {
 
   @Test(
       dataProviderClass = AdsDataProvider.class,
       dataProvider = "adDriverForcedStatusSuccess",
-      groups = {"TestAdDriverForcedStatusSuccess_GeoEdgeFree", "Ads"}
+      groups = {
+          "AdsAdDriverForcedStatusOasis",
+          "Ads"
```

Comment:
Consider removing `Ads` group and then you can write `groups = "AdsAdDriverForcedStatusOasis"`